user-guide/Advanced_Modules/Dashboards_app/Using_dashboard_feeds/Specifying_data_input_in_a_dashboard_URL.md

```diff
@@ -6,64 +6,117 @@ uid: Specifying_data_input_in_a_dashboard_URL
 
-Within the dashboard URL, the following objects can be specified:
-
-- Elements (using the argument *elements*): By specifying the DMA ID and element ID.
-
-- Services (using the argument *services*): By specifying the DMA ID and service ID.
-
-- Redundancy groups (using the argument *redundancy groups*): By specifying the DMA ID and redundancy group ID.
+> [!NOTE]
+> From DataMiner 10.2.0/10.2.2 onwards, when a dashboard updates its own URL, it will use a compressed JSON syntax. In this compressed syntax, the query parameter “d” is used instead of “data”.
 
-- Parameters (using the argument *parameters*):
+## JSON syntax
 
-  - By protocol: By specifying the protocol name, protocol version and protocol ID.
+From DataMiner 10.2.0/10.2.2 onwards, you can pass the data using a JSON object in the URL:
 
-  - By element: By specifying the DMA ID, the element ID, the parameter ID and optionally the parameter index.
+``url?data=<URL-encoded JSON object>``
 
-- Views (using the argument *views*): By specifying the view ID.
+This JSON object has to have the following structure:
 
-- SLAs (using the argument *slas*): By specifying the DMA ID and element ID.
+```json
+{
+"version": 1,
+"feedAndSelect": <data>, (optional)
+"feed": <data>, (optional)
+"select": <data>, (optional)
+"components": <component-data>
+}
+```
 
-- DataMiner Agents (using the argument *agents*): By specifying the DMA ID.
+- ``<data>`` is a JSON object with a number of property keys (corresponding with the objects listed below) and property values (as an array of strings). For example:
 
-- Protocols (using the argument *protocols*): By specifying the protocol name and protocol version.
+  ```json
+  {
+  "parameters": ["1/2/3", "1/4/6"],
+  "elements": ["1/2", "1/8", "212/123"]
+  ...
+  }
+  ```
 
-- Data Display pages:
+- When you provide data in the (optional) *feedAndSelect* item, that data will be interpreted as if it was passed using the legacy syntax described below.
 
-  - By protocol (using the argument *protocol pages*): By specifying the protocol name, protocol version and page name.
+- When you provide data in the (optional) *feed* item, that data will only be used in the URL feed. It will not be used to select items in selection boxes on the dashboard.
 
-  - By element (using the argument *parameter pages*): By specifying the DMA ID, the element ID and the page name.
+- When you provide data in the (optional) *select* item, that data will only be used to select items in selection boxes on the dashboard. It will not be used in the URL feed.
 
-- Indices (using the argument *indices*): By specifying the index.
+- In the *components* item, you can provide data to be selected in specific components referred to by their ID. ``<component-data>`` is an array consisting of the component ID and the data that should be passed to the component:
```

Comment:
Would be a bit more clear to write '...is an array of objects containing the...'